src/page/GroupEdit.tsx

```diff
@@ -35,20 +34,22 @@ const GroupEdit = () => {
     intro: introduce || '',
     group: information || '',
     previewImage: imageSrc || null,
+    selectedTags: tags || '',
   };
 
   const [inputs, setInputs] = useState(initialData);
   const [isPublicGroup, setIsPublicGroup] = useState<boolean>(!password);
   const [previewImage, setPreviewImage] = useState<string | null>(
-    imageSrc || ''
+    initialData.previewImage
   );
   const [selectedImageFile, setSelectedImageFile] = useState<File | null>(null);
   const [selectedTags, setSelectedTags] = useState<string[]>(
-    tags?.length === DUMMY.length ? [ALL_TAG] : tags || []
+    initialData.selectedTags
   );
 
   const [savedSecretKey, setSavedSecretKey] = useState<string>(password || '');
   const navigate = useNavigate();
+  const { patchMutation } = usePatchRooms();
```

Comment:
P2: 위에서 리뷰한 내용 반영하신다면 이런 식으로 사용할 수 있어요! 
++ useGetDetaile 부분은 아래와 같이 변경

```suggestion
  const { data, isLoading: isGetDetailLoading } = useGetDetail(parseInt(id));
  const { patchMutation, isPending } = usePatchRooms();
  
  const isLoading = isPending || isGetDetailLoading;
```

Reply:
하단 return 부분을 아래와 같이 isLoading 값에 따라 분기처리 해준다면, 최종적으로 사용자는 로딩이 걸리는 모든 부분에서 로딩 스피너를 보게되어 사용자 경험이 저하되는걸 방지할 수 있어요 !

```typescript
return (
    <PageLayout category="그룹">
      {isLoading ? (
        <LoadingPage isPageLoading={true} />
      ) : (
        <Form>
          <Header>그룹 생성하기</Header>
          <Borderline />
          <GroupSetting
            isPublicGroup={isPublicGroup}
            handleActiveChange={handleActiveChange}
            handlePasswordChange={handleChangeInputs}
            secretKey={secretKey}
          />
          <ImageSection
            previewImage={previewImage}
            handleImageChange={handleImageChange}
          />
          <TitleSection
            titleValue={title}
            recruitedValue={num.toString()}
            handleMemberCountChange={handleChangeInputs}
          />
          <LanguageSection
            selectedTags={selectedTags}
            setSelectedTags={setSelectedTags}
          />
          <IntroSection
            introValue={intro}
            handleChangeTextarea={handleChangeInputs}
          />
          <ProgressSection
            progressValue={group}
            handleChangeTextarea={handleChangeInputs}
          />
          <ProfileButton>
            <CancelButton type="button" onClick={handleCancelBtnClick}>
              취소하기
            </CancelButton>
            <CommonButton
              isActive={isActive}
              category="Profile_save"
              onClick={handleSaveBtnClick}
            />
          </ProfileButton>
        </Form>
      )}
    </PageLayout>
  );
```